data/werkk.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "werkk freelancing platform",
+  "establishedAt": 1663987020,
+  "location": "Fully remote",
+  "website": "http://werkk.tech/",
+  "email": "werkk@noditt.tech",
+  "phone": "+977-9828104810",
+  "socials": {
+    "facebook": "https://www.facebook.com/werkk.tech",
+    "facebook_group": "https://www.facebook.com/groups/werkk",
```

Comment:
Sorry but please remove `facebook_group` key.
Also even if the other social keys have null value, still include those 🙏

Reply:
sure !